src/randoop/FailureSet.java

```diff
@@ -6,35 +6,55 @@
 import java.util.Set;
 
 import randoop.operation.MethodCall;
-import randoop.operation.Operation;
 import randoop.sequence.ExecutableSequence;
 import randoop.sequence.MutableSequence;
 import randoop.sequence.MutableVariable;
+import randoop.sequence.Statement;
 import randoop.sequence.Variable;
 
+/**
+ * FailureSet represents a set of failures that occurred in the execution of a particular
+ * {@link ExecutableSequence}. 
+ *
+ */
 public class FailureSet {
   
   private Set<Failure> failures = new LinkedHashSet<Failure>();
   
+  /**
+   * FailureSet.Failure represents an execution failure and holds the
+   * statement and class where the failure occurred.
```

Comment:
I don't think you mean the class where the failure occurred in the same sense as the statement where the failure occurred.  (Zeugma is good in fiction prose but bad in technical writing!)  Is it the class in which the failure occurred, or a class that represents the type of failure?